apps/admin/backend/src/app.ts

```diff
@@ -286,20 +296,21 @@ function buildApi({
     },
 
     async getSystemSettings(): Promise<SystemSettings> {
+      const electionId = store.getCurrentElectionId();
+      if (!electionId) {
+        return DEFAULT_SYSTEM_SETTINGS;
+      }
+
       try {
-        const settings = store.getSystemSettings();
-        await logger.log(
-          LogEventId.SystemSettingsRetrieved,
-          (await getUserRole()) ?? 'unknown',
-          { disposition: 'success' }
-        );
-        return settings ?? DEFAULT_SYSTEM_SETTINGS;
+        const settings = store.getSystemSettings(electionId);
+        await logger.log(LogEventId.SystemSettingsRetrieved, 'system', {
```

Comment:
I took the liberty of switching `userRole ?? 'unknown'` to `'system'` for the logging user role. The way that this is used in the app is as a system call, not something triggered by a particular user.

Reply:
I actually meant all the logging in this endpoint

Reply:
I caught that, I'm just particularly annoyed with the failure logging. Planning to remove both.